alerts/alert.css

```diff
@@ -0,0 +1,49 @@
+:root{
+    --component-background-color:whitesmoke;
+    --primary:rgb(78, 179, 145);
+    --secondary:white;
+    --success:powderblue;
+    --warning:rgb(235, 235, 72);
+    --danger:rgb(233, 76, 76);
+}
+
+#alerts{
+    margin:0 auto;    
+    max-width:950px;
+
+}
+
+.components{
+    border-radius:0.5rem;
+    background-color:var(--component-background-color);
+    padding: 0.5rem;
+}
+
+.alert{    
+    padding:0.5rem;
+    border-radius: 5px;
```

Comment:
you can use px below 10 but be consistent don't use 0. rem then

Reply:
i hv made changes